packages/aws-cdk/lib/init.ts

```diff
@@ -247,10 +257,17 @@ export async function printAvailableTemplates(language?: string) {
   }
 }
 
-export async function initializeProject(template: InitTemplate, language: string, canUseNetwork: boolean, generateOnly: boolean, workDir: string) {
+async function initializeProject(
```

Comment:
non binding comment: a lot of these functions have a ton of props, and given that its experimental why not change it so that it takes in a property bag `initializeProjectOptions` so that we can easily add properties to it in the future

Reply:
Good call out! I'll make this change in a future PR.